### PR TITLE
#121 galery improvements

### PR DESCRIPTION
**Pull request by baxgas**

### CODE REVIEW

File: app/components/RadioPlayer.vue [hunk 1]
```diff
@@ -113,7 +113,7 @@
         </div>
 
         <div class="sand-clock">
-          <IconSandclock :progress="time_percent" :live="np.live.is_live" />
+          <IconSandclock :progress="time_percent" :live="np.live.is_live.length ? true : false" />
```
Comment (gammaw) [inline app/components/RadioPlayer.vue:116]: @baxgas why is this?

**Review by gammaw — APPROVED** [review]
Thanks for this too! I tested locally and scanned through the code changes. I discovered some improvements, e.g., the arrow navigation within the modal carousel view, very nice!, but it would help if you could highlight the changes in the description of the PR (like you often do), especially if the changes affect multiple files and add a lot of code. 

Also, I guess we should start documenting components for what they are doing, who is using them etc. Do you know of any best practice for that in these type of applications, e.g., a comment in the first line of the component file, in a given format? For example, `ListeItem`, `PictureItem`, `ModalGallery`, if you're not the implementor, then it may not be trivial to see what the hierarchy is. 

A specific question: why is `ModalGallery` not under `components/gallery`?

**Comment by baxgas** [comment]
Nice catch! Since it is reworked, gallery specific modal it could be under
gallery/modal and GalleryModal component name
The sandclock i found that it didn' catch the is_live state, sofix is for
that

**Comment by baxgas** [comment]
@gammaw please review again, I implemented the new functions and there was also a merge conflict with master that needed resolving

**Review by gammaw — APPROVED** [review]
Seems all good. Thanks for the structure update and the descr -> caption fix!

**Pull request merged**